src/test/java/Pages/LoginPage.java

```diff
@@ -0,0 +1,47 @@
+package Pages;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.openqa.selenium.interactions.Actions;
+import java.time.Duration;
+
+public class LoginPage extends BasePage{
+        WebDriver driver;
+       WebDriverWait wait;
```

Comment:
**Please, remove code redundancies**

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )